src/lfo.rs

```diff
@@ -0,0 +1,100 @@
+use std::f32::consts::PI;
+
+// Premake the LFO sine, and use mod frequency as a phase value to get_frac values
```

Comment:
This is very nice! I see you implemented different wave shapes. This should also be useful for our semester project. Get work!